ui/app/forms/form.ts

```diff
@@ -29,18 +31,27 @@ export default class Form {
     }
     // to ease migration from Ember Data Models, return a proxy that forwards get/set to the data object for form field props
     // this allows for form field properties to be accessed directly on the class rather than form.data.someField
+    const proxyTarget = (target: this, prop: string) => {
+      // check if the property that is being accessed is a form field
+      let formFields = Array.isArray(target.formFields) ? target.formFields : [];
+      // in the case of formFieldGroups we need extract the fields out into a flat array
+      if (Array.isArray(target.formFieldGroups)) {
+        formFields = target.formFieldGroups.reduce((arr: FormField[], group) => {
+          const values = Object.values(group)[0] || [];
+          return [...arr, ...values];
+        }, []);
+      }
```

Comment:
This allows for the same proxying of get/set to the data object when the fields are defined as `formFieldGroups`.